### ISSUE TITLE
Ill vertical alignment

### ISSUE THREAD

**Issue by memeplex** [domtronn/all-the-icons.el#287]
When using all-the-icons with company-box and doom-modeline I'm seeing lots of alignment problems, some of which are really bad:

* company-box:

![image](https://user-images.githubusercontent.com/2845433/136617936-61b8436f-5bbb-4941-8bbf-188fad329fe1.png)

The icons in `company-box-icons-all-the-icons` come annotated with `(raise -0.24)`, which is probably related to the issue, but this is not something company-box is doing AFAICS, but the way they are returned, already instrumented that way, by all-the-icons.

* doom-modeline: see my report here https://github.com/seagle0128/doom-modeline/issues/474

**Comment by memeplex** [comment]
I can fix the company-box issue by setting `all-the-icons-default-adjust` to 0 (default was -0.2). That's fine but it's still a global setting and I'm not sure I'm not breaking anything else, so what's the rationale to set it at -0.2 by default? Is it in order to compensate the default scale factor of 1.2? If that's the case it's not working properly.

OTOH the issue with doom modeline wasn't fixed by changing the default adjust. I have a couple of doubts then: is that first file icon coming from all-the-icons? If it is, then is it unaffected by `all-the-icons-default-adjust` because doom-modeline is doing its own thing on top of this?

**Comment by seagle0128** [comment]
In fact, both in `all-the-icons` and `doom-modeline`, there are many hardcoded values for `:v-adjust`. It's hard to adjust all icons.

**Comment by memeplex** [comment]
Yes, I know it's a PITA. I hope with svg support enabled by default the community will move to svg icons.

**Comment by memeplex** [comment]
Yes, I known it's a PITA. I hope with svg enabled by default in emacs different projects in the community may agree on at least one consistent svg icon set that can be included as part of all-the-icons.
